src/helpers/formatDisbursements.ts

```diff
@@ -12,6 +12,8 @@ export const formatDisbursement = (
   id: disbursement.id,
   name: disbursement.name,
   createdAt: disbursement.created_at,
+  createdBy: `${disbursement.created_by.first_name} ${disbursement.created_by.last_name}`,
+  startedBy: `${disbursement.started_by.first_name} ${disbursement.started_by.last_name}`,
```

Comment:
If `createdBy` and `startedBy` are optional, should we check if `disbursement.created_by.first_name` and `disbursement.created_by.last_name` (same for `started_by`) are set and only then assign the value? If they are not always returned from the backend, this code will break.

Something like:
```typescript
disbursement.created_by?.first_name && disbursement.created_by?.last_name ? `${disbursement.created_by?.first_name} ${disbursement.created_by?.last_name}` : null
```